w/Whitespace.ws

```diff
@@ -1,26 +1,23 @@
-push 0   
-push 10  	 	 
-push 33  	    	
-push 100  		  	  
-push 108  		 		  
-push 114  			  	 
-push 111  		 				
-push 119  			 			
-push 32  	     
-push 44  	 		  
-copy 3	  		
-copy 6	  		 
-dup 
-push 101   		  	 	
-push 72  	  	   
-
-label 0   
-outchar	
-dup   
-push 0    
-swap 
-sub		  	
-jn	0	  
-exit
+
```

Comment:
This makes the file completely empty. I'm not sure this is intended.

Reply:
No.Whitespace language write only with space and tab.other symbols are just comment out.
you can try this code on online interpreters

Reply:
Ah my text editor removed spaces.fixed